linux_docker_resources/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN if test ${PLATFORM} = x86; then curl --silent https://packagecloud.io/gpg.ke
 RUN apt-get update && apt-get install -y build-essential ccache cmake pkg-config python3-empy python3-setuptools python3-vcstool
 
 # Install build and test dependencies of ROS 2 packages.
-RUN apt-get update && apt-get install -y clang-format-3.8 cppcheck git pydocstyle pyflakes python3-coverage python3-mock python3-nose python3-pep8 uncrustify
+RUN apt-get update && apt-get install -y clang-format-3.8 cppcheck git python3-coverage python3-mock python3-nose uncrustify
```

Comment:
I can see the motivation for using apt instead of pip. I have just had apt suggest installing `flake8` despite already having flake8 installed through pip.
If we want to stick with packages installed through apt, in the case of flake8 we'll need to either pin the [batch job script to use v 2.x](https://github.com/ros2/ci/pull/3/commits/9dfd52c9b4f4883a35ef9131db1b99b2362942a0#diff-a7a7037d3453364950fd80f5f91ed7eeR48) again, or support 2.x and 3.x simultaneously. (apt will install 2.5.4 while pip will install 3.0.4)